src/test/csi-qingcloud/README.md

```diff
@@ -40,6 +40,13 @@ Chart Version | Snapshot CRDs Version | Min K8s Version
  &lt;= 1.2.8 | only v1beta1 | 1.14 
  &gt;= 1.2.9 | both v1beta1 and v1, only v1 | 1.17 
 
+## Vertical Pod Autoscaler
+Vertical Pod Autoscaler (VPA) frees the users from necessity of setting up-to-date resource limits and requests for the containers in their pods.
+Set `enableVPA` to `true` in ` values.yaml ` to apply VerticalPodAutoscaler for csi-qingcloud-controller. (Need to make sure that [vertical-pod-autoscaler](https://github.com/kubernetes/autoscaler/tree/master/vertical-pod-autoscaler) has been installed first.)
+Set the `minAllowed` and `maxAllowed` values of each container in value.yaml.
+VPA will try to cap recommendations between min and max of limit ranges. If limit range conflicts and VPA resource policy conflict then VPA will follow VPA policy (and set values outside limit range).
+If the replicas value is 1, need to manually restart to apply recommended values.
```

Comment:
can you make it work even replicas == 1? Or add a parameter in values.yaml to control the behaviour. User hates all manual works.

Reply:
I tried all updateMode, It seems that it doesn't work when replicas ==1.
I can add a parameter `controller_replicas` in values.yaml to controll csi-qingcloud-controller's replicas. 
But csi-qingcloud-node is a DaemonSet, vpa can't make it run automatically.

Reply:
csi-qingcloud-node is considered relatively stable on resource consumption. so we agreed to leave it is.